config/datanode.example.toml

```diff
@@ -29,6 +29,9 @@ rpc_max_recv_message_size = "512MB"
 ## The maximum send message size for gRPC server.
 rpc_max_send_message_size = "512MB"
 
+## Supported compression encoding for gRPC server. Currently, `gzip` and `zstd` are supported.
+rpc_accept_compressed = ["gzip"]
```

Comment:
as discussed in slack, I think it's not necessary to configure this, especially for accepting client data